src/snyk/snykCode/views/suggestion/codeSuggestionWebviewScript.ts

```diff
@@ -281,8 +281,15 @@
     }
 
     feedbackVisibility = 'close';
-    // showCurrentFeedback();
   }
+
+  function toggleReportFpPanel() {
```

Comment:
Please consider renaming to toggleReportFalsePositivePanel for readability next time you touch that code. Fp could be a little bit too short and hard to understand in 6 months or so